code/modules/client/preference_setup/loadout/loadout_xeno/tajara.dm

```diff
@@ -381,6 +381,7 @@
 	sort_category = "Xenowear - Tajara"
 	whitelisted = list(SPECIES_TAJARA, SPECIES_TAJARA_ZHAN, SPECIES_TAJARA_MSAI)
 	allowed_roles = list("Cargo Technician", "Shaft Miner", "Quartermaster", "Head of Personnel", "Station Engineer", "Atmospheric Technician", "Chief Engineer", "Engineering Apprentice")
+	culture_restriction = list(/decl/origin_item/origin/free_council)
```

Comment:
```suggestion
	origin_restriction= list(/decl/origin_item/origin/free_council)
```
`culture_restriction` is for cultures, not origins.

Reply:
whoops